server/src/main/kotlin/com/kamelia/jellyfish/rest/auth/Routes.kt

```diff
@@ -0,0 +1,26 @@
+package com.kamelia.jellyfish.rest.auth
+
+import com.kamelia.jellyfish.core.patchOrCatch
+import com.kamelia.jellyfish.core.postOrCatch
+import com.kamelia.jellyfish.util.jwt
+import com.kamelia.jellyfish.util.respond
+import io.ktor.server.application.call
+import io.ktor.server.auth.authenticate
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.route
+
+fun Route.authRoutes() = route("/auth") {
+    login()
+
+    authenticate("auth-jwt") {
+        refresh()
+    }
```

Comment:
As stated before, another JWT verifier should be created and should protect the refresh route, as the front will send the refresh token for the authorization.

```suggestion
    authenticate("refresh-jwt") {
        refresh()
    }
```